api/package/get.js

```diff
@@ -1,9 +1,10 @@
 const S = require('fluent-schema')
-const { PackageMaintainerProps } = require('../../types/Package')
+const { PackagePublic } = require('../../types/Package')
 const { Base200WithData } = require('../../common/reply')
 
 module.exports = {
+  querystring: S.object().prop('id', S.string()).required(),
   response: {
-    200: Base200WithData('packages', S.array().items(PackageMaintainerProps))
+    200: Base200WithData('package', PackagePublic)
```

Comment:
is there a case where an authenticated maintainer might use this same route and get more private info? or is the plan for that to be a different route ?

Reply:
added this logic in the api